client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1427,6 +1428,24 @@ public void testIndexPutSettings() throws IOException {
         assertEquals(expectedParams, request.getParameters());
     }
 
+    public void testGetRepositories() {
+        Map<String, String> expectedParams = new HashMap<>();
+        expectedParams.put("master_timeout", "30s");
```

Comment:
you can use setRandomTimeout. also can we test the local flag (you can use setRandomLocal I think)

Reply:
thats awesome, i fixed the master timeout and local params by using these helpers. tyvm for showing them to me.